pkg/quarterdeck/api/v1/api.go

```diff
@@ -358,13 +359,15 @@ type OpenIDConfiguration struct {
 // ===========================================================================
 
 type User struct {
-	UserID      ulid.ULID            `json:"user_id"`
-	Name        string               `json:"name"`
-	Email       string               `json:"email"`
-	LastLogin   string               `json:"last_login"`
-	OrgID       ulid.ULID            `json:"org_id"`
-	OrgRoles    map[ulid.ULID]string `json:"org_roles"`
-	Permissions []string             `json:"permissions"`
+	UserID    ulid.ULID `json:"user_id"`
+	Name      string    `json:"name"`
+	Email     string    `json:"email"`
+	Role      string    `json:"role"`
+	LastLogin time.Time `json:"last_login"`
```

Comment:
I think I would keep orgID here. I can see why we would remove OrgRoles and Permissions.